client/keystore/src/lib.rs

```diff
@@ -317,12 +317,16 @@ impl BareCryptoStore for Store {
 #[cfg(test)]
 mod tests {
 	use super::*;
-	use tempdir::TempDir;
+	use tempfile::TempDir;
 	use primitives::{testing::{SR25519}, crypto::{Ss58Codec}};
 
+	fn tempdir_with_prefix(prefix: &str) -> TempDir {
```

Comment:
As you see is the prefix always the same, so you don't need to take it as parameter. And the actual name of the directory is not important, aka we don't need the prefix at all.

Reply:
Okay, I will fix it :)

Reply:
PTAL again